Sources/VaporTypedRoutes/RoutesBuilder+RouteContext.swift

```diff
@@ -74,7 +74,7 @@ extension RoutesBuilder {
 
         let responder = BasicResponder { request in
             if case .collect(let max) = body, request.body.data == nil {
-                return request.body.collect(max: max).flatMapThrowing { _ in
+                return request.body.collect(max: max?.value).flatMapThrowing { _ in
```

Comment:
```suggestion
                return request.body.collect(max: max?.value ?? request.application.routes.defaultMaxBodySize.value).flatMapThrowing { _ in
```

Reply:
Yeah indeed it's the proper way of fixing it.